docs/apis/inappbrowser.md

```diff
@@ -0,0 +1,330 @@
+---
+title: InAppBrowser Capacitor Plugin API
+description: The InAppBrowser Plugin provides a web browser view that allows you to load any web page externally. It behaves as a standard web browser and is useful to load untrusted content without risking your application's security.
+custom_edit_url: https://github.com/ionic-team/capacitor-os-inappbrowser/blob/main/README.md
+editApiUrl: https://github.com/ionic-team/capacitor-os-inappbrowser/blob/main/src/definitions.ts
+sidebar_label: InAppBrowser
+---
+
+# @capacitor/inappbrowser
+
+The InAppBrowser Plugin provides a web browser view that allows you to load any web page externally. It behaves as a standard web browser and is useful to load untrusted content without risking your application's security.
```

Comment:
Should we mention the 3 different ways that the plugin offers to open URLs? That is, in a WebView, in an in-app system browser (Custom Tabs and Safari), and in the device's default browser.

Reply:
I agree. I think we should include this information.